frontend/site-registry/src/app/components/common/icon.ts

```diff
@@ -38,7 +38,7 @@ import {
   FaArrowsRotate,
 } from 'react-icons/fa6';
 
-import { BsFillPinMapFill, BsEyeFill } from 'react-icons/bs';
+import { BsFillPinMapFill, BsEyeFill, BsExclamationCircle, BsExclamationTriangle } from 'react-icons/bs';
 import { BiSolidFilePdf } from 'react-icons/bi';
```

Comment:
IMO we can check if we can find something suitable in react-icons bootstrap / font awesome package

Reply:
I tried to look and found it here only. Any reason why its preferable to have it from fa package?

Reply:
Pavel refers to bootstrap / font awesome icons only . Please check it.